lti_consumer/templates/html/student.html

```diff
@@ -22,25 +27,31 @@ <h2 class="problem-header">
 
 % if launch_url and not hide_launch:
     % if launch_target in ['modal', 'new_window']:
-        <section class="wrapper-lti-link">
-            % if description:
-                <div class="lti-description">${description}</div>
-            % endif
-            <p class="lti-link external">
-                % if launch_target == 'modal':
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-modal"
-                        data-target="#${element_id + '-lti-modal'}"
-                    >
-                        ${button_text or 'View resource in a modal window'} <i class="icon fa fa-external-link"></i>
-                    </button>
-                % else:
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-new-window"
-                        data-target="${form_url}"
-                    >
-                        ${button_text or 'View resource in a new window'} <i class="icon fa fa-external-link"></i>
-                    </button>
+        <section
+            class="wrapper-lti-link"
+            aria-label="This section of the courseware is delivered by a 3rd party. As such, we cannot make any guarantees that this external resource supports the same level of accessibility."
+        >
+            % if launch_pane_html:
+                ${launch_pane_html}
```

Comment:
Note, if you do try making the template safe by default, here you would need:
```
${launch_pane_html | n, decode.utf8}
```
It wouldn't hurt making this change now, which will make future work easier.